README.md

```diff
@@ -10,43 +10,243 @@
   <img alt="License: MIT" src="https://img.shields.io/badge/License-MIT-purple.svg">
 </a>
 
-A package for building highly interactive user interfaces in pure Python inspired by
-[ReactJS](https://reactjs.org/).
+`django-idom` allows you to integrate [IDOM](https://github.com/idom-team/idom) into
+Django applications. IDOM is a pure Python library inspired by
+[ReactJS](https://reactjs.org/) for creating responsive web interfaces.
 
-**Be sure to [read the IDOM Documentation](https://idom-docs.herokuapp.com)!**
-
-If you have ideas or find a bug, be sure to post an
-[issue](https://github.com/idom-team/django-idom/issues)
-or create a
-[pull request](https://github.com/idom-team/django-idom/pulls). Thanks in advance!
-
-<h3>
-  <a
-    target="_blank"
-    href="https://mybinder.org/v2/gh/idom-team/idom-jupyter/main?filepath=notebooks%2Fintroduction.ipynb"
-  >
-    Try it Now
-    <img alt="Binder" valign="bottom" height="25px"
-    src="https://mybinder.org/badge_logo.svg"
-    />
-  </a>
-</h3>
+**You can try IDOM now in a Jupyter Notebook:**
+<a
+  target="_blank"
+  href="https://mybinder.org/v2/gh/idom-team/idom-jupyter/main?filepath=notebooks%2Fintroduction.ipynb">
+  <img
+    alt="Binder"
+    valign="bottom"
+    height="21px"
+    src="https://mybinder.org/badge_logo.svg"/>
+</a>
 
-Click the badge above to get started! It will take you to a [Jupyter Notebooks](https://jupyter.org/)
-hosted by [Binder](https://mybinder.org/) with some great examples.
 
-### Or Install it Now
+# Install Django IDOM
 
 ```bash
 pip install django-idom
 ```
 
 # Django Integration
 
-This version of IDOM can be directly integrated into Django. For example
+To integrate IDOM into your application you'll need to modify or add the following files to `your_app`:
+
+```
```

Comment:
Rename `your_app` to `your_project`, `django_project`, or `example_project` since this top level folder in this example isn't an installable app.